content/en/docs/setup/getting-started/index.md

```diff
@@ -88,6 +88,12 @@ Follow these steps to get started with Istio:
     ✔ Installation complete
     {{< /text >}}
 
+    {{< warning >}}
+    If your platform has a custom demonstration profile, e.g., Openshift, use
+    it in the following command, instead of the `demo` profile. Refer to your
```

Comment:
Oops, I noticed too late (after I approved), that the warning should either be moved ahead of the command or changed like this:
```
it in the above command, ...
```

Reply:
@frankbu Added https://github.com/istio/istio.io/pull/8464